site/_posts/2017-10-25-api-contributors-guide.md

```diff
@@ -0,0 +1,255 @@
+---
+title: API Contributor's Guide
+author: jntullo
+date: 2017-10-25
+tags: collaboration developers tutorials api
```

Comment:
We're keeping the categories/tags relatively high level, so the first 3 tags are ok (also existing tags) but maybe not api (for now). As we get more contributions, we can revisit creating new tags when we reach critical mass. Thanks!